tools/ci_build/build.py

```diff
@@ -458,6 +458,10 @@ def parse_arguments():
 
     parser.add_argument("--use_rocm", action='store_true', help="Build with ROCm")
     parser.add_argument("--rocm_home", help="Path to ROCm installation dir")
+
+    # Code coverage
+    parser.add_argument("--android_coverage", action='store_true', help="Generate Android code coverage.")
+
```

Comment:
Change this to --code_coverage? Since we do not want too many platform specific arguments
Add add a check to raise error if using --code_coverage  without --android

Reply:
Doesn't this confuse people with the code coverage independent of the plat form?